auth-web/src/components/auth/staff/continuation-application/ContinuationAuthorizationReviewResult.vue

```diff
@@ -0,0 +1,243 @@
+<template>
+  <div
+    v-if="!!continuationIn"
+    id="continuation-authorization-review-result"
+  >
+    <!-- Previous Correspondence -->
+    <section class="section-container">
+      <v-row no-gutters>
+        <v-col
+          cols="12"
+          sm="3"
+          class="pr-4"
+        >
+          <label>Previous Correspondence</label>
+        </v-col>
+
+        <v-col
+          id="previous-correspondence"
+          cols="12"
+          sm="9"
+          class="pt-4 pt-sm-0"
+        >
+          <!--
+            FUTURE: implement this (ticket #22072)
+          -->
+
+          <!-- User Submission -->
+          <div>
+            <p class="ma-0 font-weight-bold">
+              June 2, 2024 at 1:30 pm Pacific time &mdash; User Submission
+            </p>
+          </div>
+
+          <v-divider class="my-6" />
+
+          <!-- Change Requested -->
+          <div>
+            <p class="ma-0 font-weight-bold">
+              June 3, 2024 at 11:45 am Pacific time &mdash; Change Requested &mdash; Staffanie Stafford
+            </p>
+
+            <p class="mt-4 mb-0">
+              Thank you for submitting your application and the accompanying files.
+            </p>
+            <p class="mt-4 mb-0">
+              Please review the following items of concern with your application:
+            </p>
+            <p class="mt-4 mb-0">
+              We found that some of the uploaded files are not legible. Please re-upload the following
+              document(s): Home Jurisdiction Authorization.
+            </p>
+            <p class="mt-4 mb-0">
+              You can update your application by visiting [your application].
+            </p>
+          </div>
+
+          <v-divider class="my-6 " />
+
+          <!-- User Resubmission -->
+          <div>
+            <p class="ma-0 font-weight-bold">
+              June 3, 2024 at 9:20 pm Pacific time &mdash; User Resubmission
+            </p>
+          </div>
+        </v-col>
+      </v-row>
+    </section>
+
+    <v-divider class="my-6 mx-6" />
+
+    <!-- Review Result -->
+    <section
+      v-if="isActionable"
+      class="section-container pt-0"
+    >
+      <v-row no-gutters>
+        <v-col
+          cols="12"
+          sm="3"
+          class="pr-4"
+        >
+          <label>Review Result</label>
+        </v-col>
+
+        <v-col
+          cols="12"
+          sm="9"
+          class="pt-4 pt-sm-0"
+        >
+          <v-form
+            ref="form"
+            lazy-validation
+            @submit.prevent
+          >
+            <v-select
+              v-model="reviewResult"
+              filled
+              :items="reviewResultItems"
+              item-text="desc"
+              item-value="code"
+              label="Review Result"
+              :rules="reviewResultRules"
+              :menu-props="{ offsetY: true }"
+              @change="$emit('review-result', reviewResult)"
```

Comment:
Changes are passed to parent, because it's the parent that handles the actions (Submit or Cancel).